index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE HTML>
+<HTML lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./style.css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconn4gt1ect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="navbar">
+                <div class="navbar-content">
+                    <a href="#" class="logo">My Logo</a>
+                    <img src="images/button.png">
+                </div>
+               <div class="nav_link hide">
+                <ul>
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+               </div>
+        </div>
+    </header>
+    <section class="section-1">
+        <div class="section-1-container">
+                <h1 class="heading">I’m Mariane<br>Glad to see you!</h1>
+                <p class="leading">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p class="social-heading">LET’S CONNECT</p>
+                <div class="socials">
+                    <img src="images/Vector.svg">
+                    <img src="images/Linkedin icon.svg">
+                    <img src="images/Vector (2).svg">
+                    <img src="images/Vector (3).svg">
+                    <img src="images/Vector (4).svg">
+                </div>
```

Comment:
- [ ] [REQUIRED] Good job so far! The `list of icons` uses a `ul` tag to group them, and `li` per icon; each icon should have a `link` (`a` tag) 🙏🏼